NonSucking.Framework.Serialization/Serializers/ListSerializer.cs

```diff
@@ -262,21 +247,21 @@ private static ITypeSymbol GetGenericTypeOf(ITypeSymbol type, out bool isTypeGen
         {
             ITypeSymbol genericArgument = null;
             var typeForIter = type;
-            while (genericArgument == null)
+            while (genericArgument is null)
             {
                 if (typeForIter is INamedTypeSymbol nts)
                 {
 
                     if (nts.TypeArguments.Length > 0)
                     {
                         genericArgument = nts.TypeArguments[0];
-                        break;
+                        continue;
```

Comment:
Warum hast du das in ein continue geändert?

Reply:
damit die abbruch bedingung im head einen sinn hat